docs/_snippets/page-story.md

```diff
@@ -154,7 +154,11 @@ export const LoggedIn: Story = {
 };
 ```
 
-```js filename="Page.stories.js" renderer="svelte" language="js"
+```svelte filename="Page.stories.svelte" renderer="svelte" language="js" tabTitle="Svelte CSF"
+<!-- TK: vet this against recommendation and apply it to the other languages (eg., ts-4-9, ts) -->
+```
```

Comment:
note that TypeScript will complain here, because it doesn't know that this is doable, but it works fine at runtime.